### PR TITLE
Update vaults-and-strategies.md

### PR DESCRIPTION
**Pull request by philburrrt**
direct contracts to automatically updated sources

### CODE REVIEW

**Comment by lehnberg** [comment]
Ackshually... yearn-hub was __just__ relaunched as https://yearn.watch. Could you update? 😬😇

File: yearn-finance/yvaults/vaults-and-strategies.md [hunk 1]
```diff
@@ -1,62 +1,7 @@
...
-## Locked Token Vaults
-
-| Vault                     | Contract       | Strategies                 |
-|---------------------------|----------------|----------------------------|
-|yveCRV|[veCRV-DAO yVault](https://etherscan.io/address/0xc5bDdf9843308380375a611c18B50Fb9341f502A#code)||
-|yvBOOST|[Yearn Compounding veCRV yVault](https://etherscan.io/address/0x9d409a0A012CFbA9B15F6D4B36Ac57A46966Ab9a)|[StrategyYearnVECRV](https://etherscan.io/address/0x683b5C88D48FcCfB3e778FF0fA954F84cA7Ce9DF)|
-
-
-
+**Vault and strategy contracts:** [Yearn Hub](https://yearn-hub.vercel.app/)
```
Comment (lehnberg) [inline yearn-finance/yvaults/vaults-and-strategies.md:7]: yearn-hub.vercel.app → yearn.watch

**Review by lehnberg — CHANGES_REQUESTED** [review]
.

**Comment by philburrrt** [comment]
lolol fixed thx

**Review by lehnberg — APPROVED** [review]

**Comment by lehnberg** [comment]
Fixes #285

**Pull request merged**